packages/eslint-plugin/src/configs/import.ts

```diff
@@ -0,0 +1,69 @@
+export = {
```

Comment:
A docstring would be really nice here.

Reply:
Added docstrings to each of the config parts in the latest commit.